python/cloudtik/providers/_private/aws/workspace_provider.py

```diff
@@ -62,7 +62,7 @@ def subscribe_global_variables(self, cluster_config: Dict[str, Any]):
         return global_variables
 
     def validate_config(self, provider_config: Dict[str, Any]):
-        if len(self.workspace_name) > 45 or not check_workspace_name_format(self.workspace_name):
+        if len(self.workspace_name) > 31 or not check_workspace_name_format(self.workspace_name):
```

Comment:
45 -> 31 in the message.
Suggest to define a constant and use the constant instead of the number to avoid such change problems.